### PR TITLE
admin: Fix isValidSerial

### PR DESCRIPTION
**Pull request by Dezash**
This PR fixes isValidSerial matching strings that are not necessarily valid.

### CODE REVIEW

File: [admin]/admin/server/admin_server.lua [hunk 1]
```diff
@@ -286,9 +286,9 @@ function iif ( cond, arg1, arg2 )
 	return arg2
 end
 
+local serialExp = "^" .. string.rep ( "[A-F0-9]", 32 ) .. "$"
```
Comment (Citizen01) [inline [admin]/admin/server/admin_server.lua]: `[A-F0-9]` could have been `%x`
Source: https://www.lua.org/pil/20.2.html

File: [admin]/admin/server/admin_server.lua [hunk 2]
```diff
@@ -1471,8 +1471,9 @@ addEventHandler ( "aBans", _root, function ( action, data, arg1, arg2, arg3 )
 			end
 		elseif ( action == "banserial" ) then
 			mdata = data
-			if ( isValidSerial ( data ) ) then
-				local newban = addBan ( nil,nil, string.upper ( data ),source,arg2, arg3 )
+			local serial = string.upper ( data )
+			if ( isValidSerial ( serial ) ) then
+				local newban = addBan ( nil,nil, serial,source,arg2, arg3 )
```
Comment (Citizen01) [inline [admin]/admin/server/admin_server.lua]: can you add spaces after all comas to seperate the args cleanly as well ? Thanks for the PR.

File: [admin]/admin/server/admin_server.lua (hunk 1 above)
Comment (Dezash) [reply [admin]/admin/server/admin_server.lua]: @Citizen01 %x is equivalent to [0-9a-fA-F]. The idea was not to include lower case characters.

File: [admin]/admin/server/admin_server.lua (hunk 2 above)
Comment (Dezash) [reply [admin]/admin/server/admin_server.lua]: Fixed.

File: [admin]/admin/server/admin_server.lua (hunk 1 above)
Comment (Citizen01) [reply [admin]/admin/server/admin_server.lua]: seems fair.

**Comment by qaisjp** [comment]
Thanks for the PR @Dezash and thanks for the review @Citizen01

**Pull request merged**